src/Zicht/Bundle/UrlBundle/Aliasing/Aliasing.php

```diff
@@ -109,7 +115,7 @@ public static function validateInternalConflictingStrategy($conflictingInternalU
      *
      * @param string $publicUrl
      * @param bool $asObject
-     * @param null $mode
+     * @param null|integer $mode
      * @return null
```

Comment:
Didn't change anything here.  But you are right that the return type can also be string and UrlEntity -> will fix